frontend/packages/dev-console/src/components/monitoring/dashboard/MonitoringDashboardPodCount.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Bullseye } from '@patternfly/react-core';
+import { Firehose, FirehoseResult, LoadingInline } from '@console/internal/components/utils';
+import { PodModel } from '@console/internal/models';
+import './MonitoringDashboardPodCount.scss';
+
+interface MonitoringDasboardPodCountProps {
+  namespace: string;
+}
+
+interface PodCountProps {
+  pods?: FirehoseResult;
+}
+
+const PodCount: React.FC<PodCountProps> = ({ pods }) => {
+  return (
+    <div className="odc-monitoring-dashboard-pod-count">
+      <h5>Pod Count</h5>
+      <Bullseye>
+        {pods.loaded ? (
+          <h1 className="odc-monitoring-dashboard-pod-count__count">{pods.data.length}</h1>
+        ) : (
+          <LoadingInline />
```

Comment:
@vikram-raj can we have specs for conditional branch here?

Reply:
Adding test for this.

Reply:
@invincibleJai Added tests for this. PTAL.